src/structures/StoreChannel.js

```diff
@@ -7,15 +7,25 @@ const GuildChannel = require('./GuildChannel');
  * @extends {GuildChannel}
  */
 class StoreChannel extends GuildChannel {
-  _patch(data) {
-    super._patch(data);
+  /**
+   * @param {*} guild The guild the store channel is part of
+   * @param {*} data The data for the store channel
```

Comment:
The is redundant

```suggestion
   * @param {*} guild The guild store channel is part of
   * @param {*} data The data for the store channel
```

Reply:
It is not. "the" store channel refers to "this" store channel that's being instantiated.

"The guild store channel is part of" is not a valid sentence structure and grammatically wrong.

If anything the first "The" is redundant ("Guild the store channel is part of") however that's a library wide decision I followed for this PR